packages/compass-import-export/src/import/analyze-csv-fields.ts

```diff
@@ -155,6 +158,10 @@ export function analyzeCSVFields({
 
         if (!headerFields) {
           headerFields = results.meta.fields ?? [];
+          if (headerFields.length) {
+            // if the file contained a BOM it will be included at the start of the first field
+            headerFields[0] = headerFields[0].replace(/^\ufeff/, '');
+          }
```

Comment:
If we're using `strip-bom-stream` anyway, I think you could also add that at the top here similar to the Utf8Validator part

Reply:
aah I meant to go back to that and change it then forgot. Will fix, quickly.